Add namespace filtering in operator. Update CI-pipeline. Update docs (#1324)
berops/claudie: docs/getting-started/detailed-guide.md

    terraformer-fd664b7ff-dd2h7    1/1     Running     0               8m9s
    ```

    !!! warning "Changing the namespace" 

>         By default, Claudie will deploy in the `claudie` namespace, and it will watch for `Input Manifest` and provider `Secrets` in this namespace. If you would like to add additional namespace to watch - overwrite `CLAUDIE_NAMESPACES` environment variable in `claudie-operator` deployment. Example:

Reviewer: ```suggestion
        By default, Claudie will monitor all namespaces, and it will watch for `Input Manifest` and provider `Secrets` in the cluster. If you would like to limit the namespaces to watch - overwrite `CLAUDIE_NAMESPACES` environment variable in `claudie-operator` deployment. Example:
```
